cabal.project

```diff
@@ -45,6 +37,21 @@ package ouroboros-network
 package ouroboros-consensus
   tests: True
 
+package contra-tracer
+  tests: False
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-prelude
+  tag: 9352518fd9f957ab5e9611a2e27f495339c597d1
+  subdir: .
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-prelude
+  tag: 9352518fd9f957ab5e9611a2e27f495339c597d1
+  subdir: .
```

Comment:
Twice the same repo + subdir

Reply:
Good catch, I carried this from the current `cabal.project`.

Reply:
It's even repeated three times :D